lib/config-generator.js

```diff
@@ -570,17 +570,16 @@ class ConfigGenerator {
         const contentBase = pathUtil.getContentBase(this.webpackConfig);
 
         const devServerOptions = {
-            contentBase: contentBase,
-            // this doesn't appear to be necessary, but here in case
-            publicPath: this.webpackConfig.getRealPublicPath(),
+            static: {
+                directory: contentBase,
+                // this doesn't appear to be necessary, but here in case
+                publicPath: this.webpackConfig.getRealPublicPath(),
+            },
             // avoid CORS concerns trying to load things like fonts from the dev server
             headers: { 'Access-Control-Allow-Origin': '*' },
-            hot: this.webpackConfig.useHotModuleReplacementPlugin(),
             // required by FriendlyErrorsWebpackPlugin
```

Comment:
this comment should be removed, as you removed the option that was explained by the comment.

Reply:
thanks for catching that!